tmt/steps/provision/testcloud.py

```diff
@@ -1473,9 +1473,14 @@ def prepare(self, logger: tmt.log.Logger) -> None:
         logger.debug(f"Created console log directory '{self.exchange_directory}'.", level=3)
 
         self.exchange_directory.chmod(0o755)
-        self.guest._run_guest_command(
-            Command("chcon", "--type", "virt_log_t", self.exchange_directory), silent=True
-        )
+
+        output = self.guest._run_guest_command(Command('cat', '/proc/filesystems'), silent=True)
+        if output is None or output.stdout is None:
+            return None
+        if 'selinux' in output.stdout:
```

Comment:
tmt should already know whether selinux is available, see & test `self.guest.facts.has_selinux`.